spec/unit/provider/user/linux_spec.rb

```diff
@@ -34,6 +34,8 @@
     "uid" => "-u",
     "shell" => "-s",
     "password" => "-p",
+    "expire_date" => "-e",
+    "inactive" => "-f",
```

Comment:
Let's add unit tests below for these two new options, you should be able to copy the others.

Reply:
The current tests are only checking manage_home behavior. I'm not very familiar with writing these kinds of tests.

Reply:
See lines 61-73 - just tests to ensure that the right CLI options are set by default and change when people set the right resource propreties.

Reply:
unit tests added